homeassistant/components/dsmr/sensor.py

```diff
@@ -197,6 +208,15 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return the unique ID of the smart meter."""
+        # Make sure telegram contains the identifier
+        if self._identifier not in self.telegram:
+            return None
+
+        return getattr(self.telegram[self._identifier], "value", None)
```

Comment:
What kind of value does this return?

Reply:
A unique serial for the smart meter. I see that I also need to append the OBI to it which then makes for a unique identifier per entity.

However it looks like this code sets up the entities before it receives the first telegram, so the equipment serial is not available yet. Trying to figure out how to delay entity creation until the first telegram is received.